network-api/networkapi/buyersguide/templates/fragments/chapter_table_of_contents.html

```diff
@@ -0,0 +1,29 @@
+<div class="container">
+  <div class="row">
+    <div class="pt-4 pt-md-5 {% if not page.notes %}pb-4 pb-md-5{% endif %} col-lg-8 offset-lg-2 justify-content-center">
+      <div class="row mb-5 mt-0 mt-sm-5 d-flex align-items-center">
+        <div class="col-12">
+          {% for child in page.get_children %}
+            <div class="row publication-row pt-3 pb-3">
+              <div class="col-auto">
+                <div class="publication-chapter-number publication-chapter-number--small">
+                  {{ forloop.counter }}
+                </div>
+              </div>
+              <div class="col-sm col-8 d-flex align-items-center">
+                <h3 class="publication-chapter-title publication-chapter-title--small mt-0">
```

Comment:
do we have a style guide document for this that we can point to, a la https://foundation.mozilla.org/en/style-guide/ ?

Reply:
😬 I'm not sure. Who should we ask about this, Sabrina?

Reply:
@sabrinang ^^